web/templates/symptom_tracker/new.html.eex

```diff
@@ -3,7 +3,7 @@
 
   <%= form_for @changeset, symptom_tracker_path(@conn, :create), fn f -> %>
     <div class="">
-      <%= label f, "Rate how much this problem or difficulty affected you today:", class: "f5 b db mb2" %>
+      <%= label f, "Rate how much this problem or difficulty effected you today:", class: "f5 b db mb2" %>
```

Comment:
hmmm 🤔. I really wasn't sure so took her word for it. I'm glad you saw that too.